indico/modules/events/registration/fields/simple.py

```diff
@@ -371,3 +371,64 @@ def _indico_email(value):
                 raise ValidationError(_('Invalid email address'))
 
         return _indico_email
+
+
+class AccompanyingPersonsField(RegistrationFormBillableField):
+    name = 'accompanying_persons'
+    mm_field_class = fields.List
+    mm_field_kwargs = {'cls_or_instance': fields.Nested({
```

Comment:
Not sure what to think about this... I'd treat an argument with such a name as positional-only even if it technically isn't (yet?)

A nicer alternative would be to add `mm_field_args` for positional args as well

Reply:
Yeah that would be useful, since this also happens in the `AccommodationField`